features/edition-images.feature

```diff
@@ -19,4 +19,15 @@ Feature: Images tab on edit edition
     And I have the "Preview images update" permission
     And a draft document with images exists
     When I visit the images tab of the the document with images
-    Then I should see a list of the images
\ No newline at end of file
+    And I should see a list with 2 images
+
+  Scenario: Images can be deleted from the images tab
+    Given I am a writer
+    And I have the "Preview images update" permission
+    And a draft document with images exists
+    When I visit the images tab of the the document with images
+    And I should see a list with 2 images
+    And I click to delete an image
+    And I confirm the deletion
+    Then I should see a success banner
+    And I should see a list with 1 image
```

Comment:
I think it should be possible to use multiple when/then statements in sequence to describe this journey.

```suggestion
    When I visit the images tab of the the document with images
    Then I should see a list with 2 images
    When I click to delete an image
    And I confirm the deletion
    Then I should see a success banner
    And I should see a list with 1 image
```

I've seen this pattern used before when describing multi-step journeys. For example, [admin-history-pagination.feature](https://github.com/alphagov/whitehall/blob/e9996106d76a2e31da4f7d0fbc0a24de1dc250da/features/admin-history-pagination.feature) follows a similar structure:

https://github.com/alphagov/whitehall/blob/e9996106d76a2e31da4f7d0fbc0a24de1dc250da/features/admin-history-pagination.feature#L11-L18